### DISCUSSION TITLE
contributing/how_to_contribute

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#207] [User-contributed notes]
# contributing/how_to_contribute

The Godot Engine is free and open-source. Like any community-driven project, we rely on volunteer contributions. On this page we want to showcase the various ways you as users can participate - to ...

https://docs.godotengine.org/en/stable/contributing/how_to_contribute.html

**Comment by VincentSinel** [comment]
To make proposals for godot improvment :
https://godot-proposals-viewer.github.io/
